### ISSUE TITLE
Chromecast only works when using small play button on home screen of a movie in app

### ISSUE THREAD

**Issue by mLgz0rn** [jellyfin/jellyfin#6217]
**Media Info of the file**

![343eedf](https://user-images.githubusercontent.com/6141439/123147302-27d23000-d45f-11eb-93e6-f92c15220106.PNG)

It's all files that are having this issue

**Logs**

[Chromecast not working](https://pastebin.com/8t17HEcd)
[Chromecast working](https://pastebin.com/sAGbLyNn)

**FFmpeg Logs**

[https://pastebin.com/QHV0faWt](https://pastebin.com/QHV0faWt)

**Stats for Nerds Screenshots**

**Server System (please complete the following information):**
 - OS: Docker on Linux (Unraid)
 - Jellyfin Version: 1.0.7.6
 - Hardware settings & device: Intel QSV/VAAPI on Intel i9 9900K]
 - Reverse proxy: nginx
 - Other hardware notes: none
 
**Client System (please complete the following information):**
 - Device: Xiaomi mi 9t Pro
 - OS: Android
 - Client: Android, Android TV
 - Browser (if Web client): Chrome
 - Client and Browser Version: 1.0.7.6 and 91.0.4472.114

**How to reproduce.**

-> Click chromecast icon
-> Choose chromecast
-> Go to movie
-> Click play
Chromecast just keeps spinning and app shows the error "Messageplaybackerrorundefined"

![ZKn6io3 1](https://user-images.githubusercontent.com/6141439/123145565-40414b00-d45d-11eb-88cb-f64f5aa1864c.jpg)
![tRYe1b0 1](https://user-images.githubusercontent.com/6141439/123145546-3ae40080-d45d-11eb-9396-b9f5a23ffc85.jpg)

-> Click Chromecast icon
-> Choose chromecast
-> Click small play button of a movie on the home screen
Now it starts playing on the chromecast without issues

![dvvQHdj 1](https://user-images.githubusercontent.com/6141439/123145480-2a338a80-d45d-11eb-8b49-318f07cc6da2.jpg)
![3kcWhMc 1](https://user-images.githubusercontent.com/6141439/123145468-26076d00-d45d-11eb-9038-354abf9973f5.jpg)
![jWazohW 1](https://user-images.githubusercontent.com/6141439/123145428-18ea7e00-d45d-11eb-88f0-dbd7b72951f5.jpg)

**I have to note that Casting from my pc in the chrome browser has no issues with either, both options work**

**Comment by mLgz0rn** [comment]
It's also the same in the chrome browser on my pc

**Comment by eyJhb** [comment]
Unsure what this issue is about. Haven't had much issues.

Is the issue that you cannot cast a already started player to the CHromecast?

**Comment by mLgz0rn** [comment]
> Unsure what this issue is about. Haven't had much issues.
> 
> Is the issue that you cannot cast a already started player to the CHromecast?

No thats not the issue.
I have uploaded a youtube video which might make it easier to understand 
https://www.youtube.com/watch?v=9zFERH8PAxs

tldr, clicking the play button on a poster either at the home screen or within the movie or tv library will make it play on the chromecast.
Going into the movie/tv show page for the selected movie or tv show and pressing the play button within there will make the error pop up.
